react/tic-tac-toe/src/app/screens/Game/components/boardRow.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+import styles from '../styles.module.scss';
+
+import Square from './Square';
+
+export function RenderRow ({ squares, handleClick, indexes }) {
+  return (
+    <div className={styles.bordRow}>
+      {indexes.map(i =>
```

Comment:
I would verify that the indexes aren't undefined because `.map` of a object undefined break